### PR TITLE
Open Help in a new tab #907

### PR DESCRIPTION
**Pull request by ppalaga**
https://openshift.io/openshiftio/openshiftio/plan/board/detail/907

### CODE REVIEW

**Comment by joshuawilson** [comment]
@ppalaga nice use of Openshift.io, thanks :)

**Comment by mindreeper2420** [comment]
@ppalaga - it looks like you applied this to the header file that was left behind from a migration. Can you apply this change to `index.html` on line 93? You can delete `header.html` from `src/app/components` :-)

File: src/app/components/header.html [hunk 1]
```diff
@@ -70,7 +70,7 @@
               </a>
             </li>
             <li>
-              <a href="https://fabric8.io/guide/getStarted/index.html">
+              <a href="https://fabric8.io/guide/getStarted/index.html" target="_blank">
```
Comment (mindreeper2420) [inline src/app/components/header.html]: @ppalaga - it looks like you applied this to the header file that was left behind from a migration. Can you apply this change to index.html on line 93? You can delete header.html from src/app/components :-)

Comment (ppalaga) [inline src/app/components/header.html]: > Can you apply this change to index.html on line 93? 

Done in 	c065fec

> You can delete header.html from src/app/components :-)

Done in 298e86f

**Pull request merged**